### DISCUSSION TITLE
Is tracker supported

### DISCUSSION THREAD

**Discussion by marc115uk** [BottlecapDave/HomeAssistant-OctopusEnergy#299] [Q&A]
So currently using agile for electric and standard flexible for gas, both seem to work well getting the standing charge and rates. Have just switched to tracker for gas, waiting for it to go through, staying on agile for electric. Just wanting to confirm once swap is complete am I going to get the correct daily gas figures? Will it populate any new sensors I.e yesterdays gas cost, todays gas cost and tomorrows gas cost?

**Comment by BottlecapDave** [comment]
Hi there. Once swapped you should get the new rates coming through. When I originally created the gas sensors I wasn't aware of tracker tariffs, so there's no previous/next sensors for gas. I'll add it to my list and add it in the future.

**Reply by michaeljaddy** [reply]
I have just switched to gas tracker and unfortunately this integration doesent seem to pull the data through for the tracker tariff. Some Googling revealed that the tracker uses a different API to the rest of the tariffs.

**Reply by BottlecapDave** [reply]
Hi there. People have successfully connected in the past, even to the secondary API you mentioned. But I've been seeing tests failing over the last few days as if octopus have made some changes, but have yet to find the time to investigate. Do you know what tariff you're on?

**Reply by michaeljaddy** [reply]
Hi David,

Thanks for your response. I'm on Octopus Tracker November 2022 v1 for gas.

Many thanks

Mike

**Reply by BottlecapDave** [reply]
It looks like the data should be coming through. Do you have any errors in your logs? Does anything come through if you restart your HA instance or integration?

**Reply by michaeljaddy** [reply]
Hi David,

The values have started coming through now - thank you. For some reason it just took a couple of days after my switch but all looks fine now.

Many thanks

Mike

**Reply by andyvee69** [reply]
It would be good if the Electricity and Gas sensors for tracker could show previous & next day rates, rather than half hour.

Especially as Electricity pricing seems available around 11am, gas is a lot later in the day.

**Reply by BottlecapDave** [reply]
This has been mentioned in https://github.com/BottlecapDave/HomeAssistant-OctopusEnergy/issues/310 along with reasoning for existing behaviour. Please upvote so I can gage interest :)
